### ISSUE TITLE
2025-03-19, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#114]
# Today's Arxiv Papers

(1) [MusicInfuser: Making Video Diffusion Listen and Dance](http://arxiv.org/abs/2503.14505v1), 2025-03-18

(2) [Aligning Multimodal LLM with Human Preference: A Survey](http://arxiv.org/abs/2503.14504v1), 2025-03-18

(3) [Temporal Consistency for LLM Reasoning Process Error Identification](http://arxiv.org/abs/2503.14495v1), 2025-03-18

(4) [Engineering Scientific Assistants using Interactive Structured Induction of Programs](http://arxiv.org/abs/2503.14488v1), 2025-03-18

(5) [Gricean Norms as a Basis for Effective Collaboration](http://arxiv.org/abs/2503.14484v1), 2025-03-18

(6) [Creation-MMBench: Assessing Context-Aware Creative Intelligence in MLLM](http://arxiv.org/abs/2503.14478v1), 2025-03-18

(7) [Calibrating Verbal Uncertainty as a Linear Feature to Reduce Hallucinations](http://arxiv.org/abs/2503.14477v1), 2025-03-18

(8) [DAPO: An Open-Source LLM Reinforcement Learning System at Scale](http://arxiv.org/abs/2503.14476v1), 2025-03-18

(9) [EnvBench: A Benchmark for Automated Environment Setup](http://arxiv.org/abs/2503.14443v1), 2025-03-18

(10) [LLM-FE: Automated Feature Engineering for Tabular Data with LLMs as Evolutionary Optimizers](http://arxiv.org/abs/2503.14434v1), 2025-03-18

(11) [PLAY2PROMPT: Zero-shot Tool Instruction Optimization for LLM Agents via Tool Play](http://arxiv.org/abs/2503.14432v1), 2025-03-18

(12) [Unifying Text Semantics and Graph Structures for Temporal Text-attributed Graphs with Large Language Models](http://arxiv.org/abs/2503.14411v1), 2025-03-18

(13) [Large Language Models for Virtual Human Gesture Selection](http://arxiv.org/abs/2503.14408v1), 2025-03-18

(14) [From "Hallucination" to "Suture": Insights from Language Philosophy to Enhance Large Language Models](http://arxiv.org/abs/2503.14392v1), 2025-03-18

(15) [How much do LLMs learn from negative examples?](http://arxiv.org/abs/2503.14391v1), 2025-03-18

(16) [Good/Evil Reputation Judgment of Celebrities by LLMs via Retrieval Augmented Generation](http://arxiv.org/abs/2503.14382v1), 2025-03-18

(17) [On the Standard Performance Criteria for Applied Control Design: PID, MPC or Machine Learning Controller?](http://arxiv.org/abs/2503.14379v1), 2025-03-18

(18) [Impossible Videos](http://arxiv.org/abs/2503.14378v1), 2025-03-18

(19) [MANTRA: Enhancing Automated Method-Level Refactoring with Contextual RAG and Multi-Agent LLM Collaboration](http://arxiv.org/abs/2503.14340v1), 2025-03-18

(20) [DualToken: Towards Unifying Visual Understanding and Generation with Dual Visual Vocabularies](http://arxiv.org/abs/2503.14324v1), 2025-03-18

(21) [COPA: Comparing the Incomparable to Explore the Pareto Front](http://arxiv.org/abs/2503.14321v1), 2025-03-18

(22) [Tapered Off-Policy REINFORCE: Stable and efficient reinforcement learning for LLMs](http://arxiv.org/abs/2503.14286v1), 2025-03-18

(23) [XOXO: Stealthy Cross-Origin Context Poisoning Attacks against AI Coding Assistants](http://arxiv.org/abs/2503.14281v1), 2025-03-18

(24) [DARS: Dynamic Action Re-Sampling to Enhance Coding Agent Performance by Adaptive Tree Traversal](http://arxiv.org/abs/2503.14269v1), 2025-03-18

(25) [Conversational Agents as Catalysts for Critical Thinking: Challenging Social Influence in Group Decision-making](http://arxiv.org/abs/2503.14263v1), 2025-03-18

(26) [JuDGE: Benchmarking Judgment Document Generation for Chinese Legal System](http://arxiv.org/abs/2503.14258v1), 2025-03-18

(27) [InnerSelf: Designing Self-Deepfaked Voice for Emotional Well-being](http://arxiv.org/abs/2503.14257v1), 2025-03-18

(28) [Towards a Barrier-free GeoQA Portal: Natural Language Interaction with Geospatial Data Using Multi-Agent LLMs and Semantic Search](http://arxiv.org/abs/2503.14251v1), 2025-03-18

(29) [KG-IRAG: A Knowledge Graph-Based Iterative Retrieval-Augmented Generation Framework for Temporal Reasoning](http://arxiv.org/abs/2503.14234v1), 2025-03-18

(30) [CRCE: Coreference-Retention Concept Erasure in Text-to-Image Diffusion Models](http://arxiv.org/abs/2503.14232v1), 2025-03-18

(31) [Decision Tree Induction Through LLMs via Semantically-Aware Evolution](http://arxiv.org/abs/2503.14217v1), 2025-03-18

(32) [Inferring Event Descriptions from Time Series with Language Models](http://arxiv.org/abs/2503.14190v1), 2025-03-18

(33) [Towards Harmless Multimodal Assistants with Blind Preference Optimization](http://arxiv.org/abs/2503.14189v1), 2025-03-18

(34) [Can LLMs Enable Verification in Mainstream Programming?](http://arxiv.org/abs/2503.14183v1), 2025-03-18

(35) [Synthetic Clarification and Correction Dialogues about Data-Centric Tasks -- A Teacher-Student Approach](http://arxiv.org/abs/2503.14167v1), 2025-03-18

(36) [EIAD: Explainable Industrial Anomaly Detection Via Multi-Modal Large Language Models](http://arxiv.org/abs/2503.14162v1), 2025-03-18

(37) [Speculative Decoding for Verilog: Speed and Quality, All in One](http://arxiv.org/abs/2503.14153v1), 2025-03-18

(38) [Marten: Visual Question Answering with Mask Generation for Multi-modal Document Understanding](http://arxiv.org/abs/2503.14140v1), 2025-03-18

(39) [CARE: A QLoRA-Fine Tuned Multi-Domain Chatbot With Fast Learning On Minimal Hardware](http://arxiv.org/abs/2503.14136v1), 2025-03-18

(40) [Inference-Time Intervention in Large Language Models for Reliable Requirement Verification](http://arxiv.org/abs/2503.14130v1), 2025-03-18

(41) [DangerMaps: Personalized Safety Advice for Travel in Urban Environments using a Retrieval-Augmented Language Model](http://arxiv.org/abs/2503.14103v1), 2025-03-18

(42) [GenPara: Enhancing the 3D Design Editing Process by Inferring Users' Regions of Interest with Text-Conditional Shape Parameters](http://dx.doi.org/10.1145/3706598.3713502), 2025-03-18

(43) [Learning on LLM Output Signatures for gray-box LLM Behavior Analysis](http://arxiv.org/abs/2503.14043v1), 2025-03-18

(44) [Synthetic Data Generation Using Large Language Models: Advances in Text and Code](http://arxiv.org/abs/2503.14023v1), 2025-03-18

(45) [MP-GUI: Modality Perception with MLLMs for GUI Understanding](http://arxiv.org/abs/2503.14021v1), 2025-03-18

(46) [Predicting Human Choice Between Textually Described Lotteries](http://arxiv.org/abs/2503.14004v1), 2025-03-18

(47) [LLM-based Unit Test Generation for Dynamically-Typed Programs](http://arxiv.org/abs/2503.14000v1), 2025-03-18

(48) [The KoLMogorov Test: Compression by Code Generation](http://arxiv.org/abs/2503.13992v1), 2025-03-18

(49) [Empowering Smaller Models: Tuning LLaMA and Gemma with Chain-of-Thought for Ukrainian Exam Tasks](http://arxiv.org/abs/2503.13988v1), 2025-03-18

(50) [SpaceVLLM: Endowing Multimodal Large Language Model with Spatio-Temporal Video Grounding Capability](http://arxiv.org/abs/2503.13983v1), 2025-03-18

(51) [Empowering LLMs in Decision Games through Algorithmic Data Synthesis](http://arxiv.org/abs/2503.13980v1), 2025-03-18

(52) [Navigating Rifts in Human-LLM Grounding: Study and Benchmark](http://arxiv.org/abs/2503.13975v1), 2025-03-18

(53) [FlexVLN: Flexible Adaptation for Diverse Vision-and-Language Navigation Tasks](http://arxiv.org/abs/2503.13966v1), 2025-03-18

(54) [MDocAgent: A Multi-Modal Multi-Agent Framework for Document Understanding](http://arxiv.org/abs/2503.13964v1), 2025-03-18

(55) [Survey of Adversarial Robustness in Multimodal Large Language Models](http://arxiv.org/abs/2503.13962v1), 2025-03-18

(56) [Improving LLM Video Understanding with 16 Frames Per Second](http://arxiv.org/abs/2503.13956v1), 2025-03-18

(57) [ConSCompF: Consistency-focused Similarity Comparison Framework for Generative Large Language Models](http://dx.doi.org/10.1613/jair.1.17028), 2025-03-18

(58) [Where do Large Vision-Language Models Look at when Answering Questions?](http://arxiv.org/abs/2503.13891v1), 2025-03-18

(59) [MoK-RAG: Mixture of Knowledge Paths Enhanced Retrieval-Augmented Generation for Embodied AI Environments](http://arxiv.org/abs/2503.13882v1), 2025-03-18

(60) [MMR: A Large-scale Benchmark Dataset for Multi-target and Multi-granularity Reasoning Segmentation](http://arxiv.org/abs/2503.13881v1), 2025-03-18

(61) [Bridging Social Psychology and LLM Reasoning: Conflict-Aware Meta-Review Generation via Cognitive Alignment](http://arxiv.org/abs/2503.13879v1), 2025-03-18

(62) [Enabling Inclusive Systematic Reviews: Incorporating Preprint Articles with Large Language Model-Driven Evaluations](http://arxiv.org/abs/2503.13857v1), 2025-03-18

(63) [MDTeamGPT: A Self-Evolving LLM-based Multi-Agent Framework for Multi-Disciplinary Team Medical Consultation](http://arxiv.org/abs/2503.13856v1), 2025-03-18

(64) [Disentangling Fine-Tuning from Pre-Training in Visual Captioning with Hybrid Markov Logic](http://dx.doi.org/10.1109/BigData62323.2024.10825003), 2025-03-18

(65) [Causal Discovery from Data Assisted by Large Language Models](http://arxiv.org/abs/2503.13833v1), 2025-03-18

(66) [LLM-Empowered IoT for 6G Networks: Architecture, Challenges, and Solutions](http://arxiv.org/abs/2503.13819v1), 2025-03-18

(67) [Automatic MILP Model Construction for Multi-Robot Task Allocation and Scheduling Based on Large Language Models](http://arxiv.org/abs/2503.13813v1), 2025-03-18

(68) [The Empty Chair: Using LLMs to Raise Missing Perspectives in Policy Deliberations](http://arxiv.org/abs/2503.13812v1), 2025-03-18

(69) [Empowering GraphRAG with Knowledge Filtering and Integration](http://arxiv.org/abs/2503.13804v1), 2025-03-18

(70) [LED: LLM Enhanced Open-Vocabulary Object Detection without Human Curated Data Generation](http://arxiv.org/abs/2503.13794v1), 2025-03-18

(71) [Mapping the Trust Terrain: LLMs in Software Engineering -- Insights and Perspectives](http://arxiv.org/abs/2503.13793v1), 2025-03-18

(72) [Mitigating KV Cache Competition to Enhance User Experience in LLM Inference](http://arxiv.org/abs/2503.13773v1), 2025-03-17

(73) [Do Large Language Models Understand Performance Optimization?](http://arxiv.org/abs/2503.13772v1), 2025-03-17

(74) [AccelGen: Heterogeneous SLO-Guaranteed High-Throughput LLM Inference Serving for Diverse Applications](http://arxiv.org/abs/2503.13737v1), 2025-03-17

(75) [CoDet-M4: Detecting Machine-Generated Code in Multi-Lingual, Multi-Generator and Multi-Domain Settings](http://arxiv.org/abs/2503.13733v1), 2025-03-17

(76) [Atyaephyra at SemEval-2025 Task 4: Low-Rank NPO](http://arxiv.org/abs/2503.13690v1), 2025-03-17

(77) [Pensez: Less Data, Better Reasoning -- Rethinking French LLM](http://arxiv.org/abs/2503.13661v1), 2025-03-17

(78) [INPROVF: Leveraging Large Language Models to Repair High-level Robot Controllers from Assumption Violations](http://arxiv.org/abs/2503.13660v1), 2025-03-17

(79) [Why Do Multi-Agent LLM Systems Fail?](http://arxiv.org/abs/2503.13657v1), 2025-03-17

(80) [SOSecure: Safer Code Generation with RAG and StackOverflow Discussions](http://arxiv.org/abs/2503.13654v1), 2025-03-17

(81) [Omnia de EgoTempo: Benchmarking Temporal Understanding of Multi-Modal LLMs in Egocentric Videos](http://arxiv.org/abs/2503.13646v1), 2025-03-17

(82) [Evaluating Programming Language Confusion](http://arxiv.org/abs/2503.13620v1), 2025-03-17

(83) [MetaScale: Test-Time Scaling with Evolving Meta-Thoughts](http://arxiv.org/abs/2503.13447v1), 2025-03-17

(84) [Faithfulness of LLM Self-Explanations for Commonsense Tasks: Larger Is Better, and Instruction-Tuning Allows Trade-Offs but Not Pareto Dominance](http://arxiv.org/abs/2503.13445v1), 2025-03-17

(85) [VideoMind: A Chain-of-LoRA Agent for Long Video Reasoning](http://arxiv.org/abs/2503.13444v1), 2025-03-17

(86) [Unified Autoregressive Visual Generation and Understanding with Continuous Tokens](http://arxiv.org/abs/2503.13436v1), 2025-03-17

(87) [xLSTM 7B: A Recurrent LLM for Fast and Efficient Inference](http://arxiv.org/abs/2503.13427v1), 2025-03-17

(88) [A Comprehensive Survey on Multi-Agent Cooperative Decision-Making: Scenarios, Approaches, Challenges and Perspectives](http://arxiv.org/abs/2503.13415v1), 2025-03-17

(89) [DLPO: Towards a Robust, Efficient, and Generalizable Prompt Optimization Framework from a Deep-Learning Perspective](http://arxiv.org/abs/2503.13413v2), 2025-03-17

(90) [Toward Generative 6G Simulation: An Experimental Multi-Agent LLM and ns-3 Integration](http://arxiv.org/abs/2503.13402v1), 2025-03-17

(91) [Using the Tools of Cognitive Science to Understand Large Language Models at Different Levels of Analysis](http://arxiv.org/abs/2503.13401v1), 2025-03-17

(92) [MicroVQA: A Multimodal Reasoning Benchmark for Microscopy-Based Scientific Research](http://arxiv.org/abs/2503.13399v1), 2025-03-17

(93) [Cream of the Crop: Harvesting Rich, Scalable and Transferable Multi-Modal Data for Instruction Fine-Tuning](http://arxiv.org/abs/2503.13383v1), 2025-03-17

(94) [Mitigating Visual Forgetting via Take-along Visual Conditioning for Multi-modal Long CoT Reasoning](http://arxiv.org/abs/2503.13360v1), 2025-03-17

(95) [Agents Play Thousands of 3D Video Games](http://arxiv.org/abs/2503.13356v1), 2025-03-17

(96) [Valid Text-to-SQL Generation with Unification-based DeepStochLog](http://dx.doi.org/10.1007/978-3-031-71167-1_17), 2025-03-17

(97) [LearnMate: Enhancing Online Education with LLM-Powered Personalized Learning Plans and Support](http://dx.doi.org/10.1145/3706599.3719857), 2025-03-17

(98) [Reliable and Efficient Amortized Model-based Evaluation](http://arxiv.org/abs/2503.13335v1), 2025-03-17

(99) [LLM Test Generation via Iterative Hybrid Program Analysis](http://arxiv.org/abs/2503.13580v1), 2025-03-17

(100) [LEAVS: An LLM-based Labeler for Abdominal CT Supervision](http://arxiv.org/abs/2503.13330v1), 2025-03-17
